src/common-util/src/main/resources/akka-global.conf

```diff
@@ -1,40 +1,55 @@
 akka {
-    stdout-loglevel = "OFF"
-    loggers = ["akka.event.slf4j.Slf4jLogger"]
-    loglevel = "DEBUG"
-    logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
-
-    actor {
-        # for now, using java serialization
-        allow-java-serialization = true
-        warn-about-java-serializer-usage = false
-    }
+  stdout-loglevel = "OFF"
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  loglevel = "DEBUG"
+  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
+
+  actor {
+    # for now, using java serialization
+    allow-java-serialization = true
+    warn-about-java-serializer-usage = false
+  }
+
+  remote {
+    artery {
+      transport = tcp
 
-    remote {
-        artery {
-            transport = tcp
+      canonical {
+        port = 0
+        hostname = "127.0.0.1"
+      }
 
-            canonical {
-                port = 0
-                hostname = "127.0.0.1"
-            }
+      advanced {
+        # Maximum serialized message size, including header data.
+        maximum-frame-size = 256 KiB
 
-            advanced {
-                # Maximum serialized message size, including header data.
-                maximum-frame-size = 256 KiB
-            }
+        # Disable compression:
```

Comment:
After the branch has been merged, it would be good to follow how long term testing, if it shows any changes in memory usage?